lambda/lambda_function.py

```diff
@@ -7,18 +7,63 @@
 def handler(event, context):
     bucket_name = os.environ['BUCKET_NAME']
 
-    api_endpoint = os.environ['API_ENDPOINT']
+    api_endpoint_1 = os.environ['API_ENDPOINT_1']
+    api_endpoint_2 = os.environ['API_ENDPOINT_2']
+    api_endpoint_3 = os.environ['API_ENDPOINT_3']
+    api_endpoint_4 = os.environ['API_ENDPOINT_4']
+    api_endpoint_5 = os.environ['API_ENDPOINT_5']
+    api_endpoint_6 = os.environ['API_ENDPOINT_6']
```

Comment:
I think to improve this lambda should only be responsible for a single api call. So instead of invoking the lambda once to retrieve data for 6 cities, invoke the lambda 6 times to retrieve data for one city at a time!
This would mean 6 eventbridge schedules.